---
layout: paper
title: "A Scalable Strand-Specific Protocol Enabling Full-Length Total RNA Sequencing From Single Cells."
image: /assets/images/papers/34149808.jpg
authors: Simon Haile, Richard D Corbett, Veronique G LeBlanc, Lisa Wei, Stephen Pleasance, Steve Bilobram, Ka Ming Nip, Kirstin Brown, Eva Trinh, Jillian Smith, Diane L Trinh, Miruna Bala, Eric Chuah, Robin J N Coope, Richard A Moore, Andrew J Mungall, Karen L Mungall, Yongjun Zhao, Martin Hirst, Samuel Aparicio, Inanc Birol, Steven J M Jones, Marco A Marra
ref: Haile et al. 2021. Front Genet.
journal: "Frontiers in genetics <b>12</b>, 665888 (2021)"
doi: 10.3389/fgene.2021.665888
abbrev: "Front Genet <b>12</b>, 665888 (2021)"
pub_year: 2021
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="34149808" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

RNA sequencing (RNAseq) has been widely used to generate bulk gene expression measurements collected from pools of cells. Only relatively recently have single-cell RNAseq (scRNAseq) methods provided opportunities for gene expression analyses at the single-cell level, allowing researchers to study heterogeneous mixtures of cells at unprecedented resolution. Tumors tend to be composed of heterogeneous cellular mixtures and are frequently the subjects of such analyses. Extensive method developments have led to several protocols for scRNAseq but, owing to the small amounts of RNA in single cells, technical constraints have required compromises. For example, the majority of scRNAseq methods are limited to sequencing only the 3' or 5' termini of transcripts. Other protocols that facilitate full-length transcript profiling tend to capture only polyadenylated mRNAs and are generally limited to processing only 96 cells at a time. Here, we address these limitations and present a novel protocol that allows for the high-throughput sequencing of full-length, total RNA at single-cell resolution. We demonstrate that our method produced strand-specific sequencing data for both polyadenylated and non-polyadenylated transcripts, enabled the profiling of transcript regions beyond only transcript termini, and yielded data rich enough to allow identification of cell types from heterogeneous biological samples.

