---
layout: default
title: "Research"
research: true
author: Sam Aparicio
category: misc
---
{% include JB/setup %}
# Overview

Why do some cancer patients respond to treatment, while others succumb to their disease?  Why are some treatments effective initially, but fail over time?  How do cancer cells acquire the ability to spread from one part of the body to another?  These are the fundamental and unresolved questions which motivate cancer research worldwide.  Viewing cancer progression through the lens of evolutionary theory, our approach to addressing these problems centers on studying the genomes of cancer cells as fundamental units of information encoding biological properties.  We have an active research program across several interrelated major topics including: Cancer evolution, single cell genomics, mutational signatures and prediction of drug response. The primary activities in the lab consist of experimental design, large-scale cancer (epi)genomics/transcriptome data analytics, development of machine learning and Bayesian statistics methods and biological study of ovarian, breast and lymphoid cancers.

---

# <a id="evolution">Cancer evolution</a>

<img width="800" class="center" src="/research/images/cancer_evolution.png">

Our lab is motivated by studying cancer through the lens of evolution.  We are engaged in several studies that span both temporal and spatial multi-sample studies of our cancers of interest.  Observing the dynamics of genomically-defined clones reflected in timeseries biopsies of patient tumours, patient-derived xenografts, or through spreading of clones across anatomical sites is a key area of interest for our lab.  For example in recent work, we have identified clonal expansion patterns underpinning breast cancer evolution, mapped the spread of clones within the peritoneal cavity of ovarian cancers and identified reproducible clonal dynamics in patient derived breast cancer xenografts.   Our current questions relate to drug selection and the interplay between malignant cells and the tumour microenvironment.

We have a keen interest in learning fitness trajectories from timeseries study of cancer populations within controlled interventions such as CRISPR or pharmacologic methods as a means to predict response to drugs.  Using extensions of population genetics theory, we are interested in predicting how cell populations will respond in the presence of a perturbation.  This is indeed ‘hard’ and entails the need to decipher stochastic drift, clonal interaction and positive selection.  Furthermore, drug response may not be encoded in the genome, requiring dynamic state switching through the epigenome and reflected in the transcriptome.  What proportion of drug response can be explained through encoded mutations in the genome?  This remains unknown. We are pursuing integrative, multimodal molecular views over time in bulk tissues and single cells as  substrate to address this question.

**Relevant recent work:**  
[ddClone: joint statistical inference of clonal populations from single cell and bulk tumour sequencing data](/papers/28249593)  
[Clonal genotype and population structure inference from single-cell tumor sequencing](/papers/27183439)  
[Divergent modes of clonal spread and intraperitoneal mixing in high-grade serous ovarian cancer](/papers/27182968)  
[Dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution](/papers/25470049)  
[PyClone: statistical inference of clonal population structure in cancer](/papers/24633410)  

---

# <a id="single-cell">Single cell genomics of cancer</a>

<img width="900" class="center" src="/research/images/single_cell_genomics.png">

The unit of evolutionary selection in cancer is the cell.  Extraordinary progress in measurement technologies has made it possible to reliably and accurately sequence the genomes of individual cancer cells at scale.  We have recently optimized biophysical techniques and hidden Markov model approaches to ascertain highly accurate copy number profiles of thousands of cancer cells.  As such, studying the ‘population genetics’ of cancer cells is a tractable goal.  We are developing phylogenetics and fitness computational models through measuring the population dynamics of thousands of individual cancer cells across timeseries, spatial samples and in the presence of genetic and pharmacologic intervention.  These experiments and computational methods are improving our knowledge of background mutation rates, properties of positive and negative selection (not observable in bulk samples) and how phylogenetic topologies reflect the relative fitness of clones. Furthermore, as we integrate multi-modal measurements, profiling the evolving malignant population in the context of its tumour microenvironment will be a strong interest of the lab.


**Relevant recent work:**  
[Clonal Decomposition and DNA Replication States Defined by Scaled Single-Cell Genome Sequencing](papers/31730858)
[Scalable whole-genome single-cell library preparation without preamplification](/papers/28068316)  
[Robust high-performance nanoliter-volume single-cell multiple displacement amplification on planar substrates](/papers/27412862)  
[Clonal genotype and population structure inference from single-cell tumor sequencing](/papers/27183439)  
[Dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution](/papers/25470049)  

---

# <a id="mutational-signatures">Mutational signatures in DNA repair deficient cancers</a>

<img width="900" class="center" src="/research/images/mutational_signature.png">

Utilising whole tissue and our recently developed single cell genome sequencing methods, we are interested in the underlying mechanisms of copy numbner-strunctural instability in cancer. We are interested in how to optimize the computational discovery of genome-wide structural and point mutational signatures and how signatures can identify treatment opportunities for ovarian and breast cancers.  This work is being carried out at bulk and single cell resolution. We are seeking to define the role of instability in modifying local immune responses. The ultimate goal is to identify new ways of targeting the unique biology associated in breast and ovarian cancers.

**Relevant recent work:**  
[Genomic consequences of aberrant DNA repair mechanisms stratify ovarian cancer histotypes](/papers/28436987)  
[Divergent modes of clonal spread and intraperitoneal mixing in high-grade serous ovarian cancer](/papers/27182968)  
[The somatic mutation profiles of 2,433 breast cancers refines their genomic and transcriptomic landscapes](/papers/27161491)  
[The clonal and mutational evolution spectrum of primary triple-negative breast cancers](/papers/22495314)  

---

# <a id="drug-response">Therapeutic Targeting of Unstable Genomes</a>

<img width="800" class="center" src="/research/images/drug_response.png">

Using a combination of genetic screening approaches and recent insights into direct targeting of tertiary G-quadruplex structures in the genome, we are identifying new ways to help patients whose tumours do not respond or have become resistant to standard of care therapies. Our work led us to identify G-quadruplex binding small molecule drugs as synthetic lethal to tumours with loss of genome maintenance and repair functions, including homologous recombination, NHEJ, theta-mediated end joining. With support from StandUp to Cancer, we conducted a phase 1 clinical study that provided evidence of mechanism and activity (. We also identified splicing reaction modulators with potential in unstable cancers, including novel small molecule inhibitors of CDK12 and CLK2. These approaches are being extended with improved small molecule development and into early stage translation. 

**Relevant recent work:**  
[A phase 1 study of CX5461](https://clinicaltrials.gov/ct2/show/NCT02719977),
[CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours](/papers/28211448),
[Discovery of Novel CDK12 inhibitors](https://aparicio.molonc.ca/papers/30067358)

---

## Key collaborators

Vancouver:
[David Huntsman](http://molonc.bccrc.ca/huntsman-lab/), 
[Poul Sorensen](https://www.bccrc.ca/dept/mo/people/poul-sorensen),
[Carl Hansen](http://www.msl.ubc.ca/faculty/hansen), 
[Alexandre Cote-Bouchard](https://www.stat.ubc.ca/~bouchard/), 
[Marco Marra](http://medgen.med.ubc.ca/person/marco-marra/), 
[Peter Stirling](https://www.bccrc.ca/dept/tfl/labs/stirling-lab),
[Mike Underhill](https://cps.med.ubc.ca/faculty1/underhill/),
[Martin Hirst](https://hirstlab.msl.ubc.ca/),
[Andy Roth](https://www.cs.ubc.ca/people/andrew-roth),
[Karen Cheung](https://ece.ubc.ca/karen-cheung/),
[Phil Hieter](https://www.msl.ubc.ca/people/dr-phil-hieter/)


Canada:
[Lincoln Stein](https://oicr.on.ca/investigators/lincoln-stein/),
[Anne-Marie Mes-Masson](http://crchum.chumontreal.qc.ca/en/researchers/mes-masson-anne-marie), 
[Morag Park](https://www.mcgill.ca/biochemistry/about-us/department/faculty-members/park),
[Michael Pollak](https://www.mcgill.ca/expmed/dr-michael-pollak),
[Dave Cescon](https://www.uhnresearch.ca/researcher/david-cescon)

International:
[Greg Hannon](http://www.cruk.cam.ac.uk/users/greg-hannon), 
[Carlos Caldas](http://www.cruk.cam.ac.uk/research-groups/caldas-group), 
[Sohrab Shah](https://www.mskcc.org/profile/sohrab-shah),
[Joan Brugge](https://brugge.med.harvard.edu/),
[Shankar Balasubramanian](https://www.ch.cam.ac.uk/person/sb10031),
[Charlie Swanton](https://www.crick.ac.uk/research/labs/charles-swanton),
[Simon Tavare](http://stat.columbia.edu/department-directory/name/simon-tavare/),
[Bernd Bodenmiller](https://www.bodenmillerlab.com/#/),
[Johanna Joyce](https://joycelab.org/),
[Ed Boyden](http://syntheticneurobiology.org/),
[Xiaowei Zhuang](http://zhuang.harvard.edu/),
[Phil Quirke](https://www.leedsmic.nihr.ac.uk/about-us/our-team/professor-phil-quirke)

---

## Funding sources

<div class="row">
	<div class="col-4">
		<a class="logo-anchor" href="https://bccancerfoundation.com/">
			<img src="/research/images/funding/bccf-logo.svg" alt="BC Cancer Foundation" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="http://www.cihr-irsc.gc.ca/e/193.html">
			<img src="/research/images/funding/cihr_logo-en.jpg" alt="CIHR" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="http://www.cancer.ca/research">
			<img src="/research/images/funding/logo_ccs.png" alt="CCSRI" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="https://ww5.komen.org/">
			<img src="/research/images/funding/susan-g-komen-for-the-cure-logo.png" alt="Susan G Komen" width="150" style="margin:10px">
		</a>
	</div>
	<div class="col-4">
		<a class="logo-anchor" href="https://www.cancerresearchuk.org/">
			<img src="/research/images/funding/cruk-logo.svg" alt="CRUK" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="http://www.standuptocancer.org/">
			<img src="/research/images/funding/stand-up-2-cancer.png" alt="CSU2C" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="https://www.innovation.ca/">
			<img src="/research/images/funding/cfi-logo.svg" alt="CFI" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="https://www.genomecanada.ca">
			<img src="/research/images/funding/gc-logo.png" alt="Genome Canada" width="150" style="margin:10px">
		</a>
	</div>
	<div class="col-4">
		<a class="logo-anchor" href="https://www.genomebc.ca/">
			<img src="/research/images/funding/genome-bc.png" alt="Genome BC" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="https://www.canarie.ca/">
			<img src="/research/images/funding/canarie-logo.png" alt="CANARIE" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="http://www.chairs-chaires.gc.ca/home-accueil-eng.aspx">
			<img src="/research/images/funding/CRC.jpg" alt="Canada Research Chairs" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="http://www.msfhr.org/">
			<img src="/research/images/funding/MSFHR_logo.jpg" alt="Michael Smith Foundation for Health Research" width="150" style="margin:10px">
		</a>
		<a class="logo-anchor" href="http://www.tfri.ca/">
			<img src="/research/images/funding/tfri-logo.png" alt="TFRI" width="150" style="margin:10px">
		</a>
	</div>

</div>
